electron/wire.json

```
@@ -11,15 +11,15 @@
   "distDir": "wrap/dist",
   "electronDirectory": "electron",
   "enableAsar": true,
-  "environment": "internal",
+  "environment": "production",
   "legalUrl": "https://wire.com/legal/",
   "licensesUrl": "https://wire.com/legal/licenses/",
   "maximumAccounts": "3",
-  "name": "WireInternal",
-  "nameShort": "WireInternal",
+  "name": "Wire",
+  "nameShort": "wire",
   "privacyUrl": "https://wire.com/privacy/",
   "supportUrl": "https://support.wire.com",
   "updateUrl": "https://wire-app.wire.com/win/prod/",
-  "version": "3.30.0",
+  "version": "3.30.0-115ca56b.0-f44d6e50",
```

```suggestion
  "version": "3.30.0",
```
necessary for building correctly